lib/settings.js

```diff
@@ -132,6 +134,17 @@ async function updateLocationSettings (sim, bundleId, authorized) {
   }
 }
 
+async function reduceMotion (sim, reduceMotion = true) {
+  let paths = await plistPaths(sim, 'accesibilitySettings');
+  for (let i in paths) {
+    let file = paths[i];
+    let updates = {
+      ReduceMotionEnabled: reduceMotion ? 1 : 0
```

Comment:
is it numeric or boolean parameter in the actual plist?

Reply:
@mykola-mokhnach is an integer.